Orange/classification/neural.py

```diff
@@ -264,7 +264,7 @@ def __call__(self, example, result_type=Orange.core.GetValue):
         :rtype: :class:`Orange.data.Value`, 
               :class:`Orange.statistics.Distribution` or a tuple with both
         """
-        example = Orange.data.Instance(self.domain, example)
+        #example = Orange.data.Instance(self.domain, example)
```

Comment:
Well, if you propose to fix the bug by deleting the line, then delete it. Don't just comment it out for posterity to wonder what's it really all about when the answer is (may be) nothing. :smiley: